### ISSUE TITLE
State locale.present found in sls common is unavailable

### ISSUE THREAD

**Issue by pwaller** [saltstack/salt#19607]
Redux of #13295

I'm still encountering this on Debian 8.

```
# cat locale.sls 
en_GB.UTF-8:
  locale.present
# salt-call --local state.sls locale
[INFO    ] Loading fresh modules for state activity
[INFO    ] Running state [en_GB.UTF-8] at time 11:02:26.478743
[ERROR   ] State locale.present found in sls locale is unavailable

local:
----------
          ID: en_GB.UTF-8
    Function: locale.present
      Result: False
     Comment: State locale.present found in sls locale is unavailable
     Changes:   
```

**Comment by rallytime** [comment]
Thanks for re-submitting this issue. Can you please post the output of 'salt --versions-report`?

**Comment by pwaller** [comment]
```
# salt-call --versions-report
           Salt: 2014.1.13
         Python: 2.7.8 (default, Oct 18 2014, 12:50:18)
         Jinja2: 2.7.3
       M2Crypto: 0.21.1
 msgpack-python: 0.4.2
   msgpack-pure: Not Installed
       pycrypto: 2.6.1
         PyYAML: 3.11
          PyZMQ: 14.4.0
            ZMQ: 4.0.5
```

**Comment by rallytime** [comment]
Thanks for adding that information. So the issue here is that the `present` function for the locale state was added as a new feature in the 2014.7 branch and is therefore not available in the 2014.1.x releases. If you would like to use that function, you'll need to upgrade to 2014.7.0. 

I just checked the docs online, however, and we haven't added a `versionadded` tag to that function. I'll label this as a documentation fix so that is more clear. Thanks again for bringing this up!
